### DISCUSSION TITLE
초기화 버튼을 눌렀을 때 브라우저 기본 alert가 아니라 별도의 팝업(모달)이 떴으면 좋겠어요.

### DISCUSSION THREAD

**Discussion by da-in** [Yangjaecheon-Otter-Guardians/simple-thumbnail#131] [Ideas]
### 주제

Features

### Body

- 미관상 더 보기좋은 팝업이 뜨면 좋을 것 같습니다.
- 브라우저 기본 alert는 스크립트 실행을 중단시키므로 사용을 지양하면 좋을 것 같습니다.

### BDD TASK

```bash
- [ ] 초기화하기 버튼을 눌렀을 때 다시 묻는 팝업(모달)이 뜬다.
```

**Comment by caseBread** [comment]
오 네 이거 초기에 개발할때 시간이없어서 급하게 alert으로 처리했던걸로 기억합니다. 

간단하게 라이브러리를 활용하거나 고도화가 필요하다면 직접 만드는것도 좋을 것 같습니다. :)
